src/fontra/backends/designspace.py

```diff
@@ -682,14 +680,17 @@ def buildUFOLayerGlyph(
         if component.location:
             # It's a variable component
             varCoDict = {"base": component.name, "location": component.location}
-            if component.transformation != Transformation():
+            if isinstance(component.transformation, DecomposedTransform):
                 varCoDict["transformation"] = asdict(component.transformation)
             variableComponents.append(varCoDict)
         else:
             # It's a regular component
+            decomposedTransform = DecomposedTransform(
+                **asdict(component.transformation)
+            )
             pen.addComponent(
                 component.name,
-                cleanAffine(makeAffineTransform(component.transformation)),
+                cleanAffine(decomposedTransform.toTransform()),
```

Comment:
Should I rename cleanAffine? @justvanrossum

Reply:
Why?

Reply:
MakeAffineTransform is deleted. The term `affine` used only in that function. I thought something like that `normalizeFloats` but it is not important.

Reply:
Ok yeah, the name is indeed not great. It also converts it to a tuple. Maybe just `cleanupTransform()`?

Reply:
I've renamed it to `cleanupTransform()`.